src/wallet-web-api-swagger/Main.hs

```diff
@@ -43,7 +45,18 @@ import           Pos.Wallet.Web             (CAddress, CCoin, CCurrency,
                                              CUpdateInfo, CWallet, CWalletAssurance,
                                              CWalletInit, CWalletMeta, CWalletRedeem,
                                              CWalletType, SyncProgress, WalletError,
-                                             walletApi)
+                                             walletApi,
+                                             TestReset, GetWallet, GetWallets,
+                                             UpdateWallet, DeleteWallet, ImportKey,
+                                             WalletRestore, NewWallet, IsValidAddress,
+                                             GetProfile, UpdateProfile, NewPayment,
+                                             NewPaymentExt, UpdateTx, GetHistory,
+                                             SearchHistory, NextUpdate, ApplyUpdate,
+                                             RedeemADA, RedeemADAPaperVend,
+                                             ReportingInitialized, ReportingElectroncrash,
+                                             GetSlotsDuration, GetVersion, GetSyncProgress)
+
+import           Description                -- All endpoints' description imported from here.
```

Comment:
This should be a qualified import too. (Unqualified implicit imports are forbidden by the style guide.)